workflows/explore_eml/navigate_through_eml.Rmd

```diff
@@ -9,16 +9,23 @@ library(EML)
 cn <- CNode('PROD')
 adc <- dataone::getMNode(cn, 'urn:node:ARCTIC')
 ```
+
 ```{r}
-eml_raw <- rawToChar(getObject(adc,
-                               "doi:10.18739/A2FS1H"))
-eml_view(eml_raw) # you can also pass a path to an EML file
+doc <- read_eml(getObject(adc, "doi:10.18739/A2FS1H"))
+```
+
+```{r, eval = FALSE}
+View(doc)
 ```
 
-To navigate this complex structure in R, use the `@` symbol. The `@` symbol allows you to go deeper into the EML structure and to see what slots are nested within other slots. However, you have to tell R where you want to go in the structure when you use the `@` symbol. For example, if you want to go into the data set of your EML you would use the command ```eml@dataset```. If you want to go to the creators of your data set you would use ```eml@dataset@creator```. Note here that `creators` are contained within `dataset`. If you aren't sure where you want to go, hit the tab button on your keyboard after typing `@` and a list of available locations in the structure will appear (e.g., `eml@<TAB>`):
+![](../images/eml_view.png)
+
+The complex EML document is represented in R as as series of named, nested lists. We use lists all the time in R! A `data.frame` is one example of a special kind of list that we use all the time. You may be familiar with the syntax `dataframe$column_name` which allows us to select a particular column of a `data.frame`. Under the hood, a `data.frame` is a named list of vectors with the same length. You select one of those vectors using the `$` operator, which is called the "list selector operator." 
+
+Just like you navigate in a `data.frame`, you can use the `$` operator to navigate through the EML structure. The `$` operator allows you to go deeper into the EML structure and to see what slots are nested within other slots. However, you have to tell R where you want to go in the structure when you use the `$` symbol. For example, if you want to go into the data set of your EML you would use the command ```doc$dataset```. If you want to go to the creators of your data set you would use ```doc$dataset$creator```. Note here that `creators` are contained within `dataset`. If you aren't sure where you want to go, hit the tab button on your keyboard after typing `$` and a list of available locations in the structure will appear (e.g., `doc$<TAB>`):
```

Comment:
Probably best not to use the word "slots" for this, as a "slot" in R is a specific member of an S4 class (and is accessed with the `@` operator).  Maybe here this should be changed to "see what lists are nested within other lists".